nginx.conf

```diff
@@ -0,0 +1,16 @@
+server {
```

Comment:
Not sure you wanted to add that file in this one?

Reply:
I did mean to add this actually because it is necessary for MacOS docs contributors.  I've modified description above to reference the fact that this is added.

Reply:
Let's make another PR then? Have you tried the build.sh & preview.sh scripts?

Reply:
yeah - I told him to add this file (he is on a mac and it's required). See https://gitlab.com/fedora/docs/docs-website/docs-fp-o/-/blob/prod/nginx.conf

@fifofonix - can you put the addition of this file in a separate commit with appropriate commit message?

Reply:
Separate commits as requested.